app/views/early_years_payment/practitioner/claims/find_reference.html.erb

```diff
@@ -1,2 +1,45 @@
-<p class="govuk-body">find reference page goes here</p>
-<%= govuk_button_to "Continue", claim_path(current_journey_routing_name), method: :patch %>
+<% content_for(:page_title, page_title(@form.t(:question), journey: current_journey_routing_name, show_error: @form.errors.any?)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @form, url: claim_path(current_journey_routing_name, request.query_parameters), method: :patch, builder: GOVUKDesignSystemFormBuilder::FormBuilder, html: { novalidate: false } do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <%= f.hidden_field :email, value: params[:email] %>
+
+      <h1 class="govuk-heading-l">
+        <%= @form.t(:question) %>
+      <h1>
+
+      <%= f.govuk_text_field :reference_number,
+        width: 10,
+        label: {
+          text: "Claim reference number",
+          tag: "h2",
+          size: "m"
+        },
+        hint: { text: "For example, ABC1234D" } %>
+
+      <%= govuk_details(summary_text: "Where you can find your claim reference number") do %>
+        <p>
+          Check your confirmation email from the Department for Education for your claim reference number.
+        </p>
+
+        <p>
+          If you can’t find the email:
+        </p>
+
+        <%= govuk_list [
+          "check your spam or junk folder",
+          "ask your employer to confirm they have sent your application. They will have received the reference number in their confirmation email."
+        ], type: :bullet %>
+
+        <p>
+          Contact us at <%= govuk_link_to "help@opsteam.com", "mailto:help@opsteam.com" %> for assistance.
```

Comment:
```suggestion
          Contact us at <%= govuk_mail_to "help@opsteam.com" %> for assistance.
```
also, is it worth putting this email in config/locales? we have the emails for the provider journey in there.

Reply:
yep agreed, i've stuck this on the last commit